### DISCUSSION TITLE
How to add visible but disabled choices?

### DISCUSSION THREAD

**Discussion by Skyofflad** [inkle/ink#823] [Writing in ink] [answered]
Hello! I have a question regarding dialogue options.
In a lot of games, when you lack a skill or a key item, certain dialogue options will be visible, but disabled.
For now, I can only change their color with tags.

**Comment by joningold** [comment, marked as answer]
Give them a tag called unclickable, and then in the code, make that option
unclickable. (You’ll have to do some coding for this, but it should be
fairly simple!) --

--
Jon Ingold
Narrative Director, *inkle*
@joningold, @inkleStudios
http://www.inklestudios.com

**Reply by Skyofflad** [reply]
Will this structure work?
`* Option 1 # {condition: enabled|disabled}`

**Comment by joningold** [comment]
Yup, it will.
-- 

--
Jon Ingold
Narrative Director, *inkle*
@joningold, @inkleStudios
http://www.inklestudios.com

**Reply by Skyofflad** [reply]
Thank you very much
